docs/howto/openstack/neutron/vpnaas.md

```diff
@@ -0,0 +1,741 @@
+    | Project                  | dfc700467396428bacba4376e72cc3e9                   |
+    | Route Mode               | static                                             |
+    | State                    | True                                               |
+    | Status                   | ACTIVE                                             |
+    | VPN Service              | d74d51f0-182d-4d88-952a-1d593ce696fd               |
+    | dpd                      | {'action': 'hold', 'interval': 30, 'timeout': 120} |
+    | project_id               | dfc700467396428bacba4376e72cc3e9                   |
+    +--------------------------+----------------------------------------------------+
+    ```
+
+## Testing the site-to-site VPN connection
+
+One way to test the VPN connection is to have two servers (e.g., 
+`server-fra1` and `server-kna1`), each on a different region (e.g., 
+`fra1` and `kna1` respectively), ping each other using private IP 
+addresses. With no VPN connection between the two regions, pinging 
+should not be possible:
+
+```console
+ubuntu@server-fra1:~$ ping -c 3 10.15.20.148
+PING 10.15.20.148 (10.15.20.148) 56(84) bytes of data.
+
+--- 10.15.20.148 ping statistics ---
+3 packets transmitted, 0 received, 100% packet loss, time 2050ms
+```
+
+```console
+ubuntu@server-kna1:~$ ping -c 3 10.15.25.58
+PING 10.15.25.58 (10.15.25.58) 56(84) bytes of data.
+
+--- 10.15.25.58 ping statistics ---
+3 packets transmitted, 0 received, 100% packet loss, time 2045ms
+```
+
+On the other hand, with a VPN connection established between the two 
+regions, pinging should be all possible:
+
+```console
+ubuntu@server-fra1:~$ ping -c 3 10.15.20.148
+PING 10.15.20.148 (10.15.20.148) 56(84) bytes of data.
+64 bytes from 10.15.20.148: icmp_seq=1 ttl=62 time=32.8 ms
+64 bytes from 10.15.20.148: icmp_seq=2 ttl=62 time=32.5 ms
+64 bytes from 10.15.20.148: icmp_seq=3 ttl=62 time=32.6 ms
+
+--- 10.15.20.148 ping statistics ---
+3 packets transmitted, 3 received, 0% packet loss, time 2003ms
+rtt min/avg/max/mdev = 32.544/32.624/32.761/0.096 ms
+```
+
+```console
+ubuntu@server-kna1:~$ ping -c 3 10.15.25.58
+PING 10.15.25.58 (10.15.25.58) 56(84) bytes of data.
+64 bytes from 10.15.25.58: icmp_seq=1 ttl=62 time=33.3 ms
+64 bytes from 10.15.25.58: icmp_seq=2 ttl=62 time=32.5 ms
+64 bytes from 10.15.25.58: icmp_seq=3 ttl=62 time=32.6 ms
+
+--- 10.15.25.58 ping statistics ---
+3 packets transmitted, 3 received, 0% packet loss, time 2003ms
+rtt min/avg/max/mdev = 32.533/32.832/33.336/0.358 ms
+```
+
+## Disabling an active connection
+
+You may, at any time, disable an active site-to-site VPN connection.
+As soon as you disable such a connection, any two servers residing in
+different regions of the previously established connection will be
+unable to reach each other by private IP addresses. You will be able
+to verify that, e.g., by pinging, as we demonstrate in the section
+regarding testing a site-to-site connection.
```

Comment:
**No.** Don't make readers jump around. Just restate the `ping` command, *at the end* of this section. The connection is down per the API, you ping to verify that it is in fact down. Simple and logical.